src/three/TilesRenderer.js

```diff
@@ -590,10 +592,20 @@ export class TilesRenderer extends TilesRendererBase {
 					.then( res => res.scene );
 				break;
 
-			case 'pnts':
-				promise = Promise.resolve( new PNTSLoader( manager ).parse( buffer ).scene );
+			}
+
+			case 'pnts': {
+
+				const loader = new PNTSLoader( manager );
+				loader.workingPath = workingPath;
+				loader.fetchOptions = fetchOptions;
```

Comment:
Small nit -- I'll clean this up. But these lines are not useful for the PNTSLoader because it's not fetching anything from anywhere. It just has to parse the file content.